frontend/src/components/pages/CreatorProfile/AddProfilePictureField.tsx

```diff
@@ -22,6 +22,8 @@ interface AddProfilePictureProps {
   field: CreatorProfileProps;
   error?: boolean;
   required?: boolean;
+  setInputField: (s: string) => void;
+  mode? : string;
```

Comment:
Turn this `mode` into an enum: https://www.typescriptlang.org/docs/handbook/enums.html

Basically define an enum like:
```
enum AddProfilePictureMode {
   creatorProfile,
    bookModal
}
```

Then you can use it in if statements like `if (mode === AddProfilePictureMode.creatorProfile` and refer to the type as `mode: AddProfilePictureMode`